Source/Core/Request.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define([
+        'Core/defaultValue'
+    ], function(
+        defaultValue) {
+    "use strict";
+
+    /**
+     * Stores information for making a request using {@link RequestScheduler}.
+     *
+     * @exports Request
+     *
+     * @private
```

Comment:
Do we plan to keep `Request` and `RequestScheduler` private when they go into master?

Although their most important use for most apps will be deep inside of Cesium for things that stream, I think they will be useful as a public API for any app that wants to play nice with requests.

Perhaps the best approach is to keep them private when this is first merged into master, and then make them public when `3d-tiles` is merged in case we want to change the interface while working on 3D Tiles.  If you agree, update the 3D Tiles roadmap to add a note to make these public.